src/Examples/Akihabara.Examples.OnRawIO/Program.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.IO;
+using Akihabara.External;
+using Akihabara.Framework;
+using Akihabara.Framework.ImageFormat;
+using Akihabara.Framework.Packet;
+using Akihabara.Framework.Port;
+using UnmanageUtility;
+
+namespace Akihabara.Examples.OnRawIO
+{
+    class Program
+    {
+        const string kInputStream = "input_video";
+        const string kOutputStream = "output_video";
+
+        static void Main(string[] args)
+        {
+            if (args.Length != 3)
+            {
+                Usage();
+                return;
+            }
+            
+            int width = Int32.Parse(args[0]);
+            int height = Int32.Parse(args[1]);
+            string configText = File.ReadAllText(args[2]);
+
+            Glog.Initialize("stuff", "stuff");
+
+            Status runStatus = RunGraph(width, height, configText);
+
+            if (!runStatus.ok)
```

Comment:
Shit, we forgot this one... It should be Status.Ok but we forgot to rename it in Akihabara.

For now it'll do, i guess